R/unnest-longer.R

```diff
@@ -20,38 +22,41 @@
 #'   column names. The default, `NULL`, gives the output columns the same names
 #'   as the input columns.
 #' @param indices_to A string giving the column name (or names) to store the
-#'   the inner names or positions (if not named) of the values. If multiple
```

Comment:
I just fixed a typo of duplicated `the the` split over these two lines. No other changes besides a reflow of the comment